packages/kubernetes-1.20/kubelet-config

```diff
@@ -74,7 +74,10 @@ systemReserved:
   {{@key}}: "{{this}}"
   {{~/each}}
 {{~/if}}
-cpuManagerPolicy: "static"
+cpuManagerPolicy: {{default "none" settings.kubernetes.cpu-manager-policy}}
```

Comment:
Are we intentionally changing the default to "none"? I thought we would expose the setting but otherwise leave the default unchanged?

```suggestion
cpuManagerPolicy: {{default "static" settings.kubernetes.cpu-manager-policy}}
```

Reply:
We currently default it to "none" here given known incompatibilities with fluent-bit as it's deployed today. We can change the default  to `static` in future once fluent-bit bug has been fixed.

Reply:
We heard from @bcressey out-of-band that he _does_ want to change the default back to `none` so I am resolving this issue and approving.